task20/js/modules/error.js

```diff
@@ -0,0 +1,4 @@
+export const renderErrorMessage = (parent, errorMessage) => {
+    parent.textContent = errorMessage;
+  };
+  
```

Comment:
nits
I am getting an alert because the last line does not contain a line break. Please check.

Reply:
Thank you very much. Fixed.
[8b063b3](https://github.com/seeeen044/JavaScript_handson/pull/21/commits/8b063b39612da9b6f4686742583d72550c5fbaab)